### PR TITLE
Maxim/cpa 90 fix tagfield css

### PR DESCRIPTION
**Pull request by RudeMax**
Turned out to be easier then I thought, just spent some time researching

### CODE REVIEW

**Comment by RudeMax** [comment]
And also I have no idea why this PR has my brunch name haha, I had some difficulties pushing it so maybe mistake has occurred in the process

**Review by RudeAnna — APPROVED** [review]
Looks great!

**Review by KenAdams777 — APPROVED** [review]

File: src/components/PoemForm/TagsAddField/TagsAddField.js [hunk 1]
```diff
@@ -78,6 +77,7 @@ function TagsAddField(props) {
 							margin="normal"
 							variant="standard"
 							InputProps={{
+								style: { flexWrap: 'wrap'},
```
Comment (MuscularSloth) [inline src/components/PoemForm/TagsAddField/TagsAddField.js:80]: I think this should be in css file =)

**Review by MuscularSloth — APPROVED** [review]
Good job!

**Pull request merged**